for/wordpress.md

```diff
@@ -0,0 +1,193 @@
+While it's possible to start from a default Codeception configuration the WPBrowser package comes with its own CLI tool (`wpcept`) and its own bootstrap command.  
+Scaffold the tests using the latter
+
+```bash
+wpcept bootstrap
+```
+
+The command creates the `tests` directory and scaffold the `acceptance`, `functional`, `wpunit` and `unit` suites creating a `tests` sub-directory and a configuration file for each.  
+Edit the `codeception.yml` file to match your local WordPress setup and point the modules to the local WordPress installation folder, the local WordPress URL and so on.  
+If you are using the `WPLoader` module in your tests take care to create a dedicated database for it and not to use the same database the `Db` or `WPDb` modules might use.  
+The use of the modules defined in the WPBrowser package is not tied to this bootstrap though so feel free to set up Codeception in any other way.
+
+## WordPress Unit Tests
+Most references to "WordPress unit tests" imply a reference to [PhpUnit](https://phpunit.de/ "PHPUnit – The PHP Testing Framework") powered tests where, but, a local WordPress instance is loaded in the same scope as the tests before the test methods are executed.  
+So "WordPress" as WordPress is loaded before the tests and "unit tests" as the subject under test is a single class or a group of tightly related functions that can be thought about as a unit.  
+By default WPBrowser creates the `wpunit` suite to keep these tests separate from "real" unit tests stored in the `unit` suite.  
+The `wpunit` suite includes the `WPLoader` module: this takes care of loading, installing and configuring a fresh WordPress installation before each test method runs.  
+To handle the heavy lifting the module requires some information about the local WordPress installation: in the `codeception.yml` file configure it to match your local setup; with reference to the example above the module configuration might be:
+
+```yaml
+modules:
+    enabled:
+        WPLoader:
+            wpRootFolder: /var/www/wordpress
+            dbName: wordpress-tests
+            dbHost: localhost
+            dbUser: root
+            dbPassword: root
+            wpDebug: true
+            tablePrefix: wptests_
+            domain: wordpress.dev
+            plugins:
+                - acme-plugin/plugin.php
+            activatePlugins:
+                - acme-plugin/plugin.php
+```
+
+The module is wrapping and augmenting the [WordPress PHPUnit automated testing suite](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/) and to generate a test case that uses Codeception and the methods provided by the Core testing suite you can use the `wpcept` command again:
+
+```bash
+wpcept generate:wpunit wpunit "Acme\SomeClass"
+```
+
+The generated test case extends the `Codeception\TestCase\WPTestCase` class and it exposes all the mehtods defined by `Codeception\TestCase\Test` test case and the Core suite `WP_UnitTestCase`.  
```

Comment:
`Codeception\TestCase\Test` is now `Codeception\Test\Unit` (in 2.2)
there is still working alias for that class so it should work but better stick to new naming